xdk/build.gradle.kts

```diff
@@ -129,7 +130,7 @@ val compileAggregate = tasks.register<JavaExec>("compileAggregate") {
     jvmArgs("-Xms1024m", "-Xmx1024m", "-ea")
 
     classpath(javatoolsJar)
-    args("-o", "$libDir",
+    args("-o", "$libDir", "$seed",
          "-version", "$xdkVersion",
```

Comment:
Apologies that there is no compileXtc task yet, but I am working on unifying this and should be done very soon (TM) now, after having spent time trying to understand how Gradle really wants things. 

So - either way - this is the master build script for the actual XTC platform. This is the wrong place to add a source of non determinism even if it's a user defined one. 
 
We really don't want to change how we build the actual XTC platform in random ways.  This is another argument that seed is something we add from the unit tests. Even if the unit tests are meta enough to just create the XTC platform with random seeds.

Reply:
That is a good point